docs/sources/developers/http_api/dashboard_public.md

```diff
@@ -20,7 +20,7 @@ title: Public Dashboard HTTP API
 
 {{% admonition type="note" %}}
 
-If you're running Grafana Enterprise, you'll need to have specific permissions for some endpoints. Refer to [Role-based access control permissions]({{< relref "../../administration/roles-and-permissions/access-control/custom-role-actions-scopes" >}}) for more information.
+If you're running Grafana Enterprise, you'll need to have specific permissions for some endpoints. Refer to [Role-based access control permissions][] for more information.
```

Comment:
No link?

Reply:
Link destination is https://github.com/grafana/grafana/pull/75875/files#diff-cfed1c3498a8f56a2eb0e2022859bec521ffee1259cc8df5532bc9ca4d59d4fdR322-R323.

It's a Markdown reference-style link rather than an inline link.